django_fido/models.py

```diff
@@ -91,3 +103,76 @@ def attestation(self) -> AttestationObject:
     def attestation(self, value: AttestationObject):
         self.attestation_data = base64.b64encode(value).decode('utf-8')
         self.credential_id_data = base64.b64encode(value.auth_data.credential_data.credential_id).decode('utf-8')
+
+    @property
+    def key_identification(self) -> Optional[str]:
+        """Return key identification.
+
+        Depending on protocol used to register the key, this can be:
+            aaid
+            aaguid
+            attestationCertificateKeyIdentifiers
+        """
+        # First test the presence of aaguid - FIDO 2
+        if self.attestation.auth_data.credential_data.aaguid != NULL_AAGUID:
+            return str(UUID(b2a_hex(self.credential.aaguid).decode()))
+        else:
+            # FIXME: Add handling for UAF devices with AAID
+            # Get the certificate FIDO U2F
+            if 'x5c' in self.attestation.att_statement:
+                cert = self.attestation.att_statement['x5c'][0]
+                certificate = load_der_x509_certificate(cert, default_backend())
+            else:
+                # ECDSAA attestation or self attestation?
+                return None
+            try:
+                subject_identifier = certificate.extensions.get_extension_for_oid(ExtensionOID.SUBJECT_KEY_IDENTIFIER)
+            except ExtensionNotFound:
+                return None
+            return b2a_hex(subject_identifier.value.digest).decode()
+
+    @property
+    def level(self) -> Optional[AuthLevel]:
+        """Return certification level."""
+        if self.key_identification is None:
+            # We cannot get anything out of the metadata
+            return None
```

Comment:
Isn't `AuthLevel.NONE` for this purpose?

Reply:
I wanted to distinguish between published metadata without certification and no metadata available.